actors/market/tests/verify_deals_for_activation_test.rs

```diff
@@ -164,7 +164,7 @@ fn verification_and_weights_for_verified_and_unverified_deals() {
             data: verified_deal_1.piece_cid,
             size: verified_deal_1.piece_size,
         },
-        s_response.activated.get(0).unwrap()
```

Comment:
Clippy does not like `get(0)`.

Reply:
`.first()` or `.front()` is

> easier to read and has the same result

but, that's just like your opinion, man

Reply:
I generally agree. I switched it to indexing (`[0]`) where it made sense.